js/index.js

```diff
@@ -40,3 +40,75 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
```

Comment:
It is great that you provided comments. It really helps with readability, especially for this project.